core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -452,6 +458,9 @@
       "The time to wait for a tablet server to process a bulk import request."),
   TSERV_MINTHREADS("tserver.server.threads.minimum", "20", PropertyType.COUNT,
       "The minimum number of threads to use to handle incoming requests."),
+  TSERV_MINTHREADS_ALLOW_TIMEOUT("tserver.server.thread.timeout.allowed", "false",
+      PropertyType.BOOLEAN,
+      "True if the incoming request threads are allowed to timeout with no work available."),
```

Comment:
minor nit: I think this is a great set of features; however, I had a question about the wording. "True... if... allowed" made me think another setting would impact this, but I imagine it is the case that if this is true then timeouts can occur. 

"Allows request threads to time out when no work is available" might be more clear but @keith-turner spent more time here so he can likely correct me from his perspective.

Reply:
I have no problem changing the descriptions as such.  done.